.gitignore

```diff
@@ -1,4 +1,4 @@
-
+Dockerfile
```

Comment:
I don't know if this should be here. Maybe add it to your private .git/info/exclude list.